packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -72,20 +70,20 @@ const getCidFromV0Transaction = (txResponse: TransactionResponse): CID => {
 }
 
 const getCidFromV1Transaction = (txResponse: TransactionResponse): CID => {
-  const decodedArgs = iface.decodeFunctionData('anchorDagCbor', txResponse.data)
-  const rootCID = decodedArgs[0]
-  const multihash = createMultihash(SHA256_CODE, uint8arrays.fromString(rootCID.slice(2), 'base16'))
+  // function signatures are encoded as 0x + 8 chars, the rest is our bytes32 param
+  const rootMultihash = txResponse.data.slice(10)
+  const multihash = createMultihash(SHA256_CODE, uint8arrays.fromString(rootMultihash, 'base16'))
   return CID.create(1, DAG_CBOR_CODE, multihash)
 }
 
 /**
  * Parses the transaction data to recover the CID.
- * @param version version of the anchor proof. Version 1 anchor proofs are created using the official anchoring smart contract and must be parsed accordingly
+ * @param txType transaction type of the anchor proof. Currently support `raw` and `dagCbor(bytes32)`
```

Comment:
is it `dagCbor(bytes32)` or `f(bytes32)`?

Reply:
Sorry, this should say `f(bytes32)`.